### PR TITLE
Fix a few typos/inconsitencies in recent tests

### PR DESCRIPTION
**Pull request by swissspidy**
Trac ticket: 

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: tests/phpunit/tests/functions/wpParseSlugList.php [hunk 1]
```diff
@@ -7,7 +7,7 @@
  *
  * @covers ::wp_parse_slug_list
  */
-class Tests_functions_wpParseSlugList extends WP_UnitTestCase {
+class Tests_Functions_wpParseSlugList extends WP_UnitTestCase {
```
Comment (costdev) [inline tests/phpunit/tests/functions/wpParseSlugList.php]: Let's also remove the `@ticket` from the data provider.

File: tests/phpunit/tests/formatting/maybeHashHexColor.php [hunk 2]
```diff
@@ -7,12 +7,15 @@
...
-class Tests_formating_maybeHashHexColor extends WP_UnitTestCase {
+class Tests_Formatting_maybeHashHexColor extends WP_UnitTestCase {
 
 	/**
 	 * @ticket 60272
 	 *
 	 * @dataProvider date_sanitize_hex_color_no_hash
+	 *
+	 * @param string $color    Color.
+	 * @param string $expected Expected.
```
Comment (costdev) [inline tests/phpunit/tests/formatting/maybeHashHexColor.php:18]: The data provider docblock should be:

```php
/**
 * Data provider.
 *
 * @return array[]
 */
```

🔢 May apply to multiple files in the PR.

File: tests/phpunit/tests/functions/mceSetDirection.php [hunk 3]
```diff
@@ -7,7 +7,7 @@
  *
  * @covers ::_mce_set_direction
  */
-class Tests_functions_mceSetDirection extends WP_UnitTestCase {
+class Tests_Functions_mceSetDirection extends WP_UnitTestCase {
```
Comment (costdev) [inline tests/phpunit/tests/functions/mceSetDirection.php]: Multiple assertions in the test method here.

Suggest:
```php
$this->assertSameSets( $mce_init, $actual, 'An unexpected LTR result was returned.' );
// ...
$this->assertSameSets( $mce_init, $actual, 'An unexpected RTL result was returned.' );
```

File: tests/phpunit/tests/formatting/maybeHashHexColor.php [hunk 4]
```diff
@@ -7,12 +7,15 @@
  *
  * @covers ::maybe_hash_hex_color
  */
-class Tests_formating_maybeHashHexColor extends WP_UnitTestCase {
+class Tests_Formatting_maybeHashHexColor extends WP_UnitTestCase {
```
Comment (costdev) [inline tests/phpunit/tests/formatting/maybeHashHexColor.php]:
```suggestion
class Tests_Formatting_MaybeHashHexColor extends WP_UnitTestCase {
```
Test classes should be in TitleCase. [ref](https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#test-classes).

🔢 Applies to other test class names in the PR, including those beginning with `wpSomething`, which per the standard should be `WpSomething`. There's inconsistency within the test suite on this, but the standard is clear, so I think it's better to go with the standard here.

Comment (costdev) [reply tests/phpunit/tests/formatting/maybeHashHexColor.php]: Related: [56846](https://core.trac.wordpress.org/ticket/56846)

**Review by costdev — APPROVED** [review]
Looks good @swissspidy! Thanks for the additional updates!

**Comment by swissspidy** [comment]
Looks like @SergeyBiryukov just fixed some of these as well. Will rebase and commit the rest

File: tests/phpunit/tests/functions/wpListSort.php [hunk 5]
```diff
@@ -20,6 +20,11 @@ public function test_wp_list_sort( $input_list, $orderby, $order, $expected ) {
 		$this->assertSame( $expected, wp_list_sort( $input_list, $orderby, $order ) );
 	}
 
+	/**
+	 * Data provider.
```
Comment (mukeshpanchal27) [inline tests/phpunit/tests/functions/wpListSort.php:24]: Do we needs to follow pattern docblock?

`Data provider for function().`

Comment (swissspidy) [reply tests/phpunit/tests/functions/wpListSort.php:24]: Imo it‘s not necessary, especially since sometimes they are used for multiple tests.

**Comment by swissspidy** [comment]
https://core.trac.wordpress.org/changeset/57735

**Pull request closed without merging**